Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM photon:2.0 as builder
-MAINTAINER Venil Noronha <veniln@vmware.com>
-
-RUN tdnf install -y openssl-1.0.2o-3.ph2.x86_64
-RUN tdnf install -y go-1.10.3-1.ph2.x86_64
+RUN tdnf --refresh install -qy wget tar gzip git && \
```

Comment:
Do we need the `git` package?

Reply:
Yes, `go build` downloads the dependencies with `go get`, which requires `git`.